api/src/main/resources/workbench.yaml

```diff
@@ -2072,10 +2091,10 @@ definitions:
         description: Timestamp when the user requests identity verification.
       twoFactorEnabled:
         type: boolean
-      linkedNihUsername:
+      eraCommonsLinkedNihUsername:
         type: string
         description: The user's NIH username
-      linkExpireTime:
+      eraCommonsLinkExpireTime:
```

Comment:
Maybe we should include the eraCommonsCompletionTime here? We do include it for compliance training (see L2066), so it would be nice to include for consistency. (Plus, I could imagine eventually showing users the timestamp when they completed each requirement in a tooltip or similar.)